src/feature_flags.ts

```diff
@@ -7,14 +7,22 @@ export const FEATURE_FLAG_DEFINITIONS = {
       { value: true, label: "Show" },
       { value: false, label: "Hidden" }
     ]
+  },
+  MAP_PAGE: {
+    description: "Access to Map Page",
+    options: [
+      { value: true, label: "Show" },
+      { value: false, label: "Hidden" }
+    ]
   }
 } satisfies FlagDefinitionsType
 
 export const FEATURE_FLAGS: Record<
   keyof typeof FEATURE_FLAG_DEFINITIONS,
   boolean
 > = {
-  EVENT_PAGE: true
+  EVENT_PAGE: true,
+  MAP_PAGE: true
```

Comment:
this should be `false` before we are completely done with the feature

Reply:
Fixed